templates/repo/view_list.tmpl

```diff
@@ -1,4 +1,4 @@
-<table id="repo-files-table" class="ui single line table gt-mt-0" data-last-commit-loader-url="{{.LastCommitLoaderURL}}">
+<table id="repo-files-table" class="ui single line table gt-mt-0" hx-indicator="tr.notready td.message span" hx-trigger="load[this.querySelectorAll('tr.notready').length > 0]" hx-swap="morph" hx-post="{{.LastCommitLoaderURL}}">
```

Comment:
Maybe it could be simplified to:

`{{if HasNotReadItems}}hx-...hx-trigger="reload after 1sec" hx-...{{end}}`

Reply:
Let me know if I understand correctly. You are suggesting I loop over the files before templating the table and if I find a file with its commit being nil, I'll set `ctx.Data["HasNotReadItems"]` to true. Then in the template I render the htmx attributes only if `.HasNotReadItems` is true, otherwise it's the same table without any `hx-*` attributes.

Did I get that right?

Reply:
Yes.

ps: not only checking the "file", but also checking the last commit.

Reply:
Here we go https://github.com/go-gitea/gitea/pull/29544/commits/0f5d7d96301f71d56b9c1167164edae1e942bd8a